tools/test/topology/tplg-build.sh

```diff
@@ -20,7 +20,6 @@ fi
 # Simple component test cases
 # can be used on components with 1 sink and 1 source.
 SIMPLE_TESTS=(test-all test-capture test-playback)
-TONE_TEST=test-tone-playback
```

Comment:
I think that if you remove that variable, you should also change lines that use removed variable, f.e: `simple_test codec tone "SSP2-Codec" s32le SSP 2 s16le 20 16 1920000 19200000 I2S 0 TONE_TEST[@]`